Tests/Shared.Specs/BasicEquivalencySpecs.cs

```diff
@@ -3599,6 +3595,26 @@ public void When_asserting_members_from_different_enum_types_are_equivalent_by_s
             act.Should().NotThrow();
         }
 
+        [Fact]
+        public void When_asserting_members_from_different_enum_types_are_equivalent_by_value_but_comparing_by_name_it_should_throw()
```

Comment:
🤔 What are _asserting members_?

Reply:
Would you accept the change to "[...]asserting_that_members_from[...]"? BTW there are many places like this is this particular file and tens in other files :-).